spec/controllers/tournaments_controller_spec.rb

```diff
@@ -194,6 +194,29 @@
             expect(@group_stage_tournament.playoff_teams_amount)
               .to eq(create_group_tournament_data[:playoff_teams_amount])
           end
+
+          context 'playoff_teams_amount unacceptable' do
+            it 'is not a power of two' do
+              post :create, params: create_group_tournament_data.merge(playoff_teams_amount: 18)
+              expect(response).to have_http_status(:unprocessable_entity)
+              expect(deserialize_response(response).values.first.first)
```

Comment:
i would prefer referring to the error by keys instead of this, but if that's not possible i'm fine with this

Reply:
I don't know how I would do that, it might be possible

Reply:
```ruby
expect(deserialize_response(response)[:playoff_teams_amount].first)
```

Reply:
done